### ISSUE TITLE
If go back from panel 3 to panel 2 don't need to reinstantiate network

### ISSUE THREAD

**Issue by salvadord** [MetaCell/NetPyNE-UI#5]

**Comment by adrianq** [comment]
That's correct. We have made some assumptions and simplifications regarding panel 2 and 3. For instance, if you go to panel 2 and then to panel 3 the network will be reinstated (and afterwards simulated) although it is not needed. These panel transitions were performed at the end of the last project when we were running out of time. I wouldn't consider these bugs but features to be improved after next week. This task can improve panel transitions and also provide support for parallel simulation in the docker.
